crates/node/src/sync/key.rs

```diff
@@ -115,19 +142,27 @@ impl Node {
             &StreamMessage::Message {
                 sequence_id: sqx_out.next(),
                 payload: MessagePayload::KeyShare { sender_key },
+                next_nonce: sending_nonce, // the next nonce will not be used
```

Comment:
code is clear enough

```suggestion
                next_nonce: sending_nonce,
```